pkg/keystore/file/key.go

```diff
@@ -83,6 +83,7 @@ func encryptKey(k *ecdsa.PrivateKey, password string, edg keystore.EDG) ([]byte,
 		}
 		addr = a
 	case elliptic.P256():
+		//lint:ignore SA1019 to be addressed soon
 		addr = elliptic.Marshal(elliptic.P256(), k.PublicKey.X, k.PublicKey.Y)
```

Comment:
@janos this `elliptic.Marshal` has been deprecated, should it be replaced with someting like
```golang
pubKey, _ := k.ECDH()
addr = pubKey.Bytes()
```

Reply:
Hi @notanatol. Given that elliptic.P256 is used only for the libp2p and will not be used for the ethereum address that uses btcec.S256, it should be safe to use basically anything and the proposed solution should be ok. Current marshalling is not relevant to the libp2p key files as the Address field is not used for libp2p.